src/com/epam/javacourse/hometask3/Bookstore.java

```diff
@@ -0,0 +1,13 @@
+package com.epam.javacourse.hometask3;
+
+public abstract class Bookstore {
+    public static void displayBooksByAuthor(String authorName, Book[] books) {
```

Comment:
I would suggest changing the method's name to `displayBooksByAuthorsFullName` to make the code a bit more self-documented. Because it's not clear what can I use as a parameter in this name: first name, second name, or full name.
Or to add Javadoc with a clear usage purpose.